boards/nxp/tropic/firmware.prototype

```diff
@@ -0,0 +1,13 @@
+{
+    "board_id": 34,
```

Comment:
Reserved "NXP ucans32k146"             34
https://github.com/PX4/PX4-Bootloader/blob/main/board_types.txt#L63

Reply:
So this doesn't implement a px4 bootloader, the teensy bootloader is used.

I suppose I can just remove `firmware.prototype`

`if ver hwtypecmp V6XRT009000 V6XRT010000`

Needs to be removed indeed.